panic: runtime error: invalid memory address or nil pointer dereference

**General information:**

* Operating System: Ubuntu 20.04
* Terracognita version / tag:  Starting Terracognita with version v0.7.6
* Did you build Terracognita from sources or did you use the Docker image:  Binary install

**Describe the bug:**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x66ffdae]
```

**Log message**
```
ts=2022-05-23T22:44:25.334207993Z caller=import.go:175 func=provider.Import resource=aws_alb_target_group msg="importing done"
ts=2022-05-23T22:44:25.334232098Z caller=import.go:82 func=provider.Import resource=aws_alb_target_group_attachment msg="fetching the list of resources"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x66ffdae]

goroutine 1 [running]:
github.com/cycloidio/terracognita/aws.albTargetGroupAttachments({0x11ba4038, 0xc0000720b8}, 0xc000dd5180, {0x106959c7, 0xc000ba5210}, 0xc0016a5600)
	/tmp/build/6b18b934/code/aws/resources.go:498 +0x38e
github.com/cycloidio/terracognita/aws.(*aws).Resources(0xc002f12d50, {0x11ba4038, 0xc0000720b8}, {0x106959c7, 0x1f}, 0x3)
	/tmp/build/6b18b934/code/aws/provider.go:96 +0xba
github.com/cycloidio/terracognita/provider.Import({0x11ba4038, 0xc0000720b8}, {0x11dca848, 0xc000dd5180}, {0x11ba5768, 0xc001e46dc0}, {0x0, 0x0}, 0xc001e97260, {0x11a9b4c0, ...})
	/tmp/build/6b18b934/code/provider/import.go:91 +0xae3
github.com/cycloidio/terracognita/cmd.glob..func2(0x1c32f300, {0xc000bd0200, 0x8, 0x8})
	/tmp/build/6b18b934/code/cmd/aws.go:110 +0x846
github.com/spf13/cobra.(*Command).execute(0x1c32f300, {0xc000bd0180, 0x8, 0x8})
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x1c32f080)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/tmp/build/6b18b934/code/main.go:11 +0x25
```
and
```
ts=2022-05-23T22:29:48.115431118Z caller=import.go:82 func=provider.Import resource=aws_lb_target_group_attachment msg="fetching the list of resources"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x66ffdae]

goroutine 1 [running]:
github.com/cycloidio/terracognita/aws.albTargetGroupAttachments({0x11ba4038, 0xc0000720b8}, 0xc000d96680, {0x10696052, 0xc00043f290}, 0xc0031e1c00)
	/tmp/build/6b18b934/code/aws/resources.go:498 +0x38e
github.com/cycloidio/terracognita/aws.(*aws).Resources(0xc002bf00f0, {0x11ba4038, 0xc0000720b8}, {0x10696052, 0x1e}, 0x3)
	/tmp/build/6b18b934/code/aws/provider.go:96 +0xba
github.com/cycloidio/terracognita/provider.Import({0x11ba4038, 0xc0000720b8}, {0x11dca848, 0xc000d96680}, {0x11ba5768, 0xc001e2af00}, {0x0, 0x0}, 0xc001e7fa40, {0x11a9b4c0, ...})
	/tmp/build/6b18b934/code/provider/import.go:91 +0xae3
github.com/cycloidio/terracognita/cmd.glob..func2(0x1c32f300, {0xc000b466e0, 0xa, 0xa})
	/tmp/build/6b18b934/code/cmd/aws.go:110 +0x846
github.com/spf13/cobra.(*Command).execute(0x1c32f300, {0xc000b465a0, 0xa, 0xa})
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x1c32f080)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
	/tmp/build/6b18b934/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/tmp/build/6b18b934/code/main.go:11 +0x25
```

Seems related to #213

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,8 @@
   ([Issue #285](https://github.com/cycloidio/terracognita/issues/285))
 - Script to update providers now pushed the tag to the fork
   ([PR #294](https://github.com/cycloidio/terracognita/pull/294))
+- `aws_lb_target_group_attachment` was raising a nil pointer exception
+  ([Issue #297](https://github.com/cycloidio/terracognita/issues/297))
 
 ## [0.7.6] _2022-05-11_
 

--- a/aws/resources.go
+++ b/aws/resources.go
@@ -492,7 +492,7 @@ func albTargetGroupAttachments(ctx context.Context, a *aws, resourceType string,
 		for _, t := range targetHealths {
 			// As this are the required values to get the resource
 			// we validate that they are present
-			if t.Target == nil || i.TargetGroupArn == nil {
+			if t.Target == nil || t.Target.Id == nil || t.Target.Port == nil || i.TargetGroupArn == nil {
 				continue
 			}
 			r, err := initializeResource(a, fmt.Sprintf("%s_%d_%s", *t.Target.Id, *t.Target.Port, *i.TargetGroupArn), resourceType)

